src/Tests/WebApp.fs

```diff
@@ -360,14 +357,8 @@ let tests = testList "Web App Tests" [
         let site:WebAppConfig = webApp { name "app-name-that-is-longer-than-40-characters"; add_slot slot; zip_deploy "test.zip" }
         Expect.isTrue (site.CommonWebConfig.Slots.ContainsKey "warm-up") "Config should contain slot"
 
-        let createSlots() =
-            site
-            |> getResources
-            |> getResource<Arm.Web.Site>
-            |> List.filter (fun x -> x.ResourceType = Arm.Web.slots)
-
         let toExceptionMessage(ex:Exception) = ex.Message
-        let exceptionMessage = Expect.throwsC (fun () -> createSlots() |> ignore) toExceptionMessage
+        let exceptionMessage = Expect.throwsC (fun () -> getSlots site |> ignore) toExceptionMessage
```

Comment:
Is there a more idiomatic way of asserting on the exception message i.e. F#/Expecto equivalent of https://fluentassertions.com/exceptions/

Reply:
not that I'm aware of :(